benchmark/queries/ldbc-sf100/shortest_path_ldbc100/q37.benchmark

```diff
@@ -0,0 +1,5 @@
+-NAME q37
+-COMPARE_RESULT 1
+-QUERY MATCH (a:Person)-[r:knows* SHORTEST 1..30]->(b:Person) WHERE a.ID = 94 RETURN COUNT(*)
```

Comment:
open an issue about failed queries. Though how did you obtain the ground truth?

Reply:
I just noticed the queries are failing. I pulled the answers from my query runs, but it looks like the current master branch is giving different results. 

I may have to run it on Neo4J to confirm the answers.